modules/update-upgrading-cli.adoc

```diff
@@ -91,7 +99,7 @@ $ oc adm upgrade channel stable-{product-version}
 +
 [IMPORTANT]
 ====
-For production clusters, you must subscribe to a `stable-\*`, `eus-*`, or `fast-*` channel.
+For production clusters, you must subscribe to a `stable-\*`, `eus-*`, `fast-*`, `fleet-approved-*`, or `ga-*` channel.
```

Comment:
I recognize that the list has been made rather verbose, but I think stable and fast will be removed from this list in 4.16.